src/extract.ts

```diff
@@ -464,6 +497,55 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
     return ret;
 }
 
+function extractJuliaBenchmarkHelper(g: JuliaBenchmarkGroup, labels: string[] = []): BenchmarkResult[] {
+    const res: BenchmarkResult[] = [];
+    for (const key in g[1].data) {
+        const value = g[1].data[key];
```

Comment:
to avoid accessing `g[1]` multiple times you can use restructuring in the function signature
```suggestion
function extractJuliaBenchmarkHelper([_, bench]: JuliaBenchmarkGroup, labels: string[] = []): BenchmarkResult[] {
    const res: BenchmarkResult[] = [];
    for (const key in bench.data) {
        const value = bench.data[key];
```